NotesBackEnd/tests/Notes.Api.Tests/Controllers/NotesControllerTest.cs

```diff
@@ -7,28 +8,30 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Web.Http;
-using Notes.Api.Tests.Utils;
+using Notes.Api.Tests.Comparers;
 
 namespace Notes.Api.Tests.Controllers
 {
     [TestFixture]
-    public class NotesControllerTest
+    internal class NotesControllerTest
     {
         private NotesController _controller;
 
         [SetUp]
         public void Init()
         {
-            _controller = new NotesController();
-            _controller.Configuration = new HttpConfiguration();
+            _controller = new NotesController
+            {
+                Configuration = new HttpConfiguration(),
+                Request = new HttpRequestMessage
+                {
+                    RequestUri = new Uri("http://test")
+                }
+            };
+
             _controller.Configuration.Routes.MapHttpRoute(
                 name: "Notes",

```

Comment:
It is a bit inconvenient that the `"Notes"` string can be found in 3 locations. Please, make it an internal constant in the `NotesController` for now (it would be `private` if was not needed in the tests or if the name itself (not that it is same everywhere) was important).